lib/node/rpc.js

```diff
@@ -2152,16 +2152,21 @@ class RPC extends RPCBase {
   }
 
   async estimateFee(args, help) {
-    if (help || args.length !== 1)
-      throw new RPCError(errs.MISC_ERROR, 'estimatefee nblocks');
+    if (help || args.length < 1 || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'estimatefee nblocks (covenantType)');
 
     const valid = new Validator(args);
     const blocks = valid.u32(0, 1);
+    const val = valid.str(1);
```

Comment:
If its an optional argument, there needs to be a fallback value here

Reply:
I was relying on the behavior of `estimateFee`, which accepts a similar method signature, but you're right that I should be explicit here.